Makefile

```diff
@@ -84,12 +84,12 @@ fidesops-test: compose-up
 
 compose-up:
 	@echo "Bringing up docker containers..."
-	@docker compose up -d
+	@docker compose-up -d
```

Comment:
To use the separate `docker-compose` executable (instead of the `docker compose` command built-in to newer versions of the Docker CLI), it does need to be like:
```
docker-compose up
```

Not
```
docker compose-up
```